guides/common/modules/proc_viewing-compliance-policy-statistics.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Viewing_Compliance_Policy_Statistics_{context}"]
+= Viewing Compliance Policy Statistics
+
+You can view a compliance policy dashboard to verify compliance reports of a particular policy.
+The compliance policy dashboard provides a statistical summary of compliance of hosts and the ability to view report details for each host within the scope of that policy.
+
+For all hosts which were evaluated as non-compliant, the `Failed` statistic provides a useful metric for prioritizing compliance effort.
+The hosts detected as `Never audited` should also be a priority, since their status is unknown.
```

Comment:
Fine for me; alternatively: "{Team} recommends to ...".

Reply:
Good idea.
I also have an improvement of the Team value for upstream in #2189